tests/app/celery/test_tasks.py

```diff
@@ -693,7 +701,9 @@ def test_should_not_save_email_if_restricted_service_and_invalid_email_address(
     assert notify_db_session.session.get(Notification, notification_id) is None
 
 
-def test_should_put_save_email_task_in_research_mode_queue_if_research_mode_service(mocker, notify_db_session, sample_service, sample_template):
+def test_should_put_save_email_task_in_research_mode_queue_if_research_mode_service(
+    mocker, notify_db_session, sample_service, sample_template
+):
```

Comment:
question, not an approval blocker, on line 588 in this file, the arguments for the test are broken out onto separate lines because of the trailing comma. In this test, there is trailing comma. How picky are we being about having these? I'm assuming not so much since adding commas is a lot of time without a lot of payoff, but wanted to ask.

Reply:
I much prefer trailing commas everywhere but I have no reasonable way of enforcing it. One line per parameter makes future changes trivial by not making you parse a whole line to find the changed thing. 

I was unable to find a ruff setting that enforced it and I manually did this hundreds of times in the `app/` folder when we implemented ruff. Tests were being worked in 1436, so I didn't change those. Someone could write a python script at the end of a sprint we wrap up early.